package.json

```diff
@@ -23,12 +23,24 @@
     "jquery": "^3.4.1",
     "react": "^16.4.2",
     "react-dom": "^16.4.2",
-    "react-scripts": "1.1.4",
+    "react-scripts": "^3.0.1",
     "styled-components": "^3.4.2",
     "uuid": "^3.1.0"
   },
   "devDependencies": {
     "backstopjs": "^3.8.5"
   },
-  "heroku-run-build-script": true
+  "heroku-run-build-script": true,
+  "browserslist": {
```

Comment:
Do you mind explaining this, please?

Reply:
Those are [browserlist](https://github.com/browserslist/browserslist) definition.

`create-react-app` uses `autoprefixer` to parse CSS and add vendor prefixes to rules.
`autoprefixer` changed from their own way to manage browsers to `browserslist`recently.

This is the default configs from `browserlist` + `create-react-app`.

Trying to resume:
> When compiling to production add CSS vendor prefix to browsers that have usage >0.2% , are not officially discontinued and are not opera mini (Any version).

You can read more about this in [browserlist README.md](https://github.com/browserslist/browserslist/blob/master/README.md) or in this nice article: [Outdated browser detection with Browserslist - dev.to](https://dev.to/amplifr/outdated-browser-detection-with-browserslist-10co)